app/src/main/java/com/codesoom/assignment/application/AuthenticationService.java

```diff
@@ -0,0 +1,36 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.utills.JwtUtill;
+import io.jsonwebtoken.Claims;
+import org.springframework.stereotype.Service;
+
+/**
+ * 사용자 권한을 관리합니다.
+ */
+@Service
+public class AuthenticationService {
+    private final JwtUtill jwtUtill;
+
+    public AuthenticationService(JwtUtill jwtUtill) {
+        this.jwtUtill = jwtUtill;
+    }
+
+    /**
+     * 사용자가 로그인하면 아이디를 인코딩하여 리턴합니다.
+     *
+     * @return 인코딩된 아이디
+     */
+    public String login() {
+        return jwtUtill.encode(1L);
+    }
+
+    /**
+     * 인증된 토큰을 받아 아이디를 디코딩하여 리턴합니다.
+     *
+     * @return 사용자 아이디
+     */
+    public Long parseToken(String accessToken) {
```

Comment:
짧은 메소드라도 이 값을 변경하면 안 된다는 것을 이렇게 표현해주면 좋습니다.
```suggestion
    public Long parseToken(final String accessToken) {
```